lib/data-generator.js

```diff
@@ -19,7 +20,25 @@ module.exports = function(runner, done) {
   });
 
   runner.on('test end', function(test) {
-    var error;
+    let error,
+        result = test.ctx.result;
+
+    // go to the main directory to get the assets
+    if (result) {
+      let baseline = result.baseline,
+          screenshot = result.screenshot,
+          diff = result.diff;
+
+      if (baseline) {
+        result.baseline = path.join('..', '..', baseline);
+      }
+      if (screenshot) {
+        result.screenshot = path.join('..', '..', screenshot);
```

Comment:
Let's reuse the common part by creating a variable with the path and joining with it here.